js/apps/admin-ui/test/clients/saml.spec.ts

```diff
@@ -212,18 +212,20 @@ test.describe("Clients SAML tests", () => {
   test("should check access settings", async ({ page }) => {
     const validUrl =
       "http://localhost:8180/realms/master/protocol/" + clientId + "/clients/";
-    const invalidUrlError =
+    const invalidUrlErrorRoot =
       "Client could not be updated: invalid_inputRoot URL is not a valid URL";
+    const invalidUrlErrorBase =
+      "Client could not be updated: invalid_inputBase URL is not a valid URL";
 
     await page.getByTestId("rootUrl").fill("Invalid URL");
     await saveSamlSettings(page);
-    await assertNotificationMessage(page, invalidUrlError);
+    await assertNotificationMessage(page, invalidUrlErrorRoot);
```

Comment:
Looks like this variable is only used once, so it might as well be in lined:
```suggestion
    await assertNotificationMessage(page, "Client could not be updated: invalid_inputRoot URL is not a valid URL");
```